static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -231,6 +305,88 @@ export class AccessMethod extends Component {
     )
   }
 
+  renderHarmonySelector(harmonyMethods, selectedAccessMethod) {
+    const { selectValue } = this.state
+    if (selectedAccessMethod && selectedAccessMethod.startsWith('harmony') && selectValue === '') {
+      this.setState({
+        selectValue: harmonyMethods.find(({ methodKey }) => methodKey === selectedAccessMethod).name
+      })
+    }
+
+    return (
+      <Select.Root
+        name="HarmonyMethodSelector"
+        value={selectValue}
+        onValueChange={(e) => this.handleHarmonySelection(e, harmonyMethods)}
+      >
+        <span>Service</span>
+        <Select.Trigger key="HarmonyTrigger" className="SelectTrigger">
+          <Select.Value placeholder="Choose a service">
+            {selectValue}
+          </Select.Value>
+          <Select.Icon className="SelectIcon" />
+        </Select.Trigger>
+
+        <Select.Portal>
+          <Select.Content className="SelectContent" position="popper">
+            <ScrollArea.Root className="ScrollAreaRoot" type="auto">
+              <Select.Viewport key="HarmonySelectorViewport" className="SelectViewport" asChild>
+                <ScrollArea.Viewport className="ScrollAreaViewport">
+                  {this.harmonyMethodsMapper(harmonyMethods, selectedAccessMethod)}
+                </ScrollArea.Viewport>
+              </Select.Viewport>
+              <ScrollArea.Scrollbar
+                className="ScrollAreaScrollbar"
+                orientation="vertical"
+              >
+                <ScrollArea.Thumb className="ScrollAreaThumb" />
+              </ScrollArea.Scrollbar>
+              <Select.Arrow />
+            </ScrollArea.Root>
+          </Select.Content>
+        </Select.Portal>
+      </Select.Root>
+    )
+  }
+
+  renderStep1(hasHarmony, radioList, collectionId, selectedAccessMethod) {
```

Comment:
This one I do think we should really rename to something a little more clear to what its doing

Reply:
I renamed it to `getAccessMethodTypes()`, since that's the content of the Step 1 part of the access methods panel. How does that sound?